fe/fe-core/src/main/java/com/starrocks/load/InsertOverwriteJobRunner.java

```diff
@@ -245,19 +243,16 @@ private void gc(boolean isReplay) {
             }
             Preconditions.checkState(table instanceof OlapTable);
             OlapTable targetTable = (OlapTable) table;
-            Map<Long, Set<Long>> sourceTabletIds = Maps.newHashMap();
+            Set<Tablet> sourceTablets = Sets.newHashSet();
```

Comment:
should implement `hashCode` and `equals` methods in `LakeTablet`

Reply:
ok

Reply:
done